packages/example/src/Lottie/Halloween/Balloons.tsx

```diff
@@ -0,0 +1,44 @@
+import {Lottie} from '@remotion/lottie';
+import React from 'react';
+import {interpolate, Sequence, useCurrentFrame, useVideoConfig} from 'remotion';
+import balloonsAnimation from './balloons.json';
```

Comment:
We didn't document importing from a file directly (which works fine I think). However, should we update this example to use `staticFile`?

Reply:
Yes good idea - `staticFile` has some benefits which we'd like to promote over importing directly

Reply:
Implemented that!